specification/containerregistry/data-plane/Microsoft.ContainerRegistry/preview/2019-08-15/containerregistry.json

```diff
@@ -1366,7 +1366,7 @@
         },
         "changeableAttributes": {
           "description": "Changeable attributes",
-          "$ref": "#/definitions/ChangeableAttributes"
+          "$ref": "#/definitions/RepositoryChangeableAttributes"
```

Comment:
If you're changing the attributes type for repository and manifest, would it make sense to remove the general ChangeableAttributes type and make it resource-specific everywhere it appears?